enterprise/server/util/ocicache/ocicache_test.go

```diff
@@ -92,3 +82,61 @@ func TestCacheSecret(t *testing.T) {
 		})
 	}
 }
+
+func setupTestEnv(t *testing.T) *testenv.TestEnv {
+	te := testenv.GetTestEnv(t)
+	flags.Set(t, "app.client_identity.client", interfaces.ClientIdentityApp)
+	key, err := random.RandomString(16)
+	require.NoError(t, err)
+	flags.Set(t, "app.client_identity.key", string(key))
+	require.NoError(t, err)
+	err = clientidentity.Register(te)
+	require.NoError(t, err)
+	require.NotNil(t, te.GetClientIdentityService())
+
+	_, runServer, localGRPClis := testenv.RegisterLocalGRPCServer(t, te)
+	testcache.Setup(t, te, localGRPClis)
+	go runServer()
+	return te
+}
+
+// TestManifestWrittenOnlyToAC sets the byte stream server and client to nil,
+// writes a manifest to the AC, and fetches it. If that path were to touch the CAS
+// at all, there would be an error trying to write to a nil client or server.
+func TestManifestWrittenOnlyToAC(t *testing.T) {
+	te := setupTestEnv(t)
+	te.SetByteStreamServer(nil)
+	te.SetByteStreamClient(nil)
```

Comment:
All of the `ocicache` functions being tested below are only taking an `acClient` as a param and not an `env` or a `bsClient` / `bsServer`. So it seems like what we are actually testing is that ActionCacheClient does not depend on BytestreamServer or BytestreamClient. I don't think that's something that the ocicache package needs to be concerned with. wdyt?

Reply:
(In other words, the fact that the functions only take an AC client as a param seems like enough to demonstrate that there are no CAS dependencies, so it seems like maybe we don't need an explicit test for it, and the fact that the existing tests are passing seems good enough?)

Reply:
Good point! Paranoia moved me to write this. Removed.

Reply:
`WriteBlobOrManifestToCacheAndWriter` does take a ByteStreamClient.

Reply:
ah sorry, I missed that because it was being set to `nil`. Keeping this test seems fine in that case